File: AirportManagement.API/AirportManagement.API/Data/Models/SecurityCheck.cs
using System;
namespace AirportManagement.API.Models
{
	public class SecurityCheck
	{
		public int Id { get; set; }

		public string CheckResult { get; set; }

		public string Comments { get; set; }

		public DateTime CreatedAt { get; set; }

		public DateTime UpdatedAt { get; set; }

		public int PassangerId { get; set; }
	}
}

File: AirportManagement.SPA/src/app/_models/SecurityCheck.ts
import { Passangers } from "./Passangers";

export interface SecurityCheck {
    Id: number;
    CheckResult: string;
    Comments: string;
    CreatedAt: Date;
    UpdatedAt: Date;
    PassangerId: number;
    Passangers: Passangers;
}

export interface SecurityCheckForPassanger {
    checkResult: string;
    comments: string;
    createdAt: Date;
    updatedAt: Date;
}

